pytest.ini

```diff
@@ -54,7 +54,7 @@ markers =
     yellow_squad: marker for yellow squad
 
 
-log_format = %(asctime)s - %(threadName)s - %(name)s - %(levelname)s - %(message)s
+log_format = %(asctime)s - %(threadName)s - %(name)s - %(levelname)s - %(clusterctx)s - %(message)s
```

Comment:
Why don't we want a delimiter between the log level and the cluster context?

Reply:
Because we wanted to avoid using an output like `C[0]` in **regular deployments**.

So instead of having C[0] everywhere, we would just have an empty string (Worth to mention, that I still did not write an if conditional to check whether if the deployment is multi or single cluster, but that was the intention)